app/javascript/controllers/turbo_frame_history_controller.js

```diff
@@ -2,15 +2,15 @@ import { Controller } from "@hotwired/stimulus"
 
 export default class extends Controller {
 
-  connect() {
-    document.addEventListener('turbo:before-fetch-request', this.beforeFetchRequest)
-    window.addEventListener("popstate", this.popState)
-  }
+  // connect() {
+  //   document.addEventListener('turbo:before-fetch-request', this.beforeFetchRequest)
+  //   window.addEventListener("popstate", this.popState)
+  // }
 
-  disconnect() {
-    document.removeEventListener('turbo:before-fetch-request', this.beforeFetchRequest)
-    window.removeEventListener("popstate", this.popState)
-  }
+  // disconnect() {
+  //   document.removeEventListener('turbo:before-fetch-request', this.beforeFetchRequest)
+  //   window.removeEventListener("popstate", this.popState)
+  // }
```

Comment:
This got weird, and I don't think I have context enough about when and where Turbo is refreshing to correct it.

The behavior is, when you send a message it works, and the reply starts streaming back in. Then, when the reply is done streaming, there's a turbo broadcast refresh. That fails because it somehow tries to run a GET to a nonexistent url.

Reply:
It's okay to leave it commented out. I have an open github issue with Turbo about a better solution to this.